articles/Choosing-between-Django-Flask-and-FastAPI/index.md

```diff
@@ -0,0 +1,102 @@
+- FastAPI makes it easy to build a GraphQL API with a Python library called *graphene-python*.
+
+#### Cons
+- Because FastAPI is relatively new, the community is small compared to other frameworks, and regardless of its detailed documentation, there are few external educational materials.
+
+##### FastAPI Use cases
+Fast API, which is relatively new amongst the three, works perfectly if your concern is speed. Netflix uses it for its internal crisis management. It also scales perfectly in deploying production-ready machine learning models because ML models work best in production when they are wrapped around a REST API and deployed in a microservice.
+
+Now we're going to compare Django, Flask, and FastAPI based on their *packages*, *community*, *performance*, *flexibility*, *job opening*, and *education*.
+
+**Packages**
+Among Django, Flask, and FastAPI, Django has the most packages that enable reusability of code. It is a full-stack web development framework, unlike Flask, a microservice, and FastAPI, a minimalistic framework for building fast websites.
+
+**Community**
+Django has the biggest community because of its wide use and popularity next to the Flask, which also has a thriving community. FastAPI, on the other hand, has a small community because it's fairly new.
+
+**Performance**
+In performance, FastAPI leads because it is speed-oriented, then next to Flask, and finally Django, which is not very fast.
+
+**Flexibility**
+Flexibility is something developers value a lot, and Flask is more flexible than Django. Fast API, on the other hand, is flexible code-wise and doesn't restrict the code layout. So we can say Flask is the most flexible among all of the above.
+
+**Job openings**
+No doubt they are more job openings in the python web ecosystem with Django requirement next to Flask and finally Fast API, which is few, so if you aim to get a quick job, then Django should be your go-to guy.
+
+**Education**
+Django is more strenuous to learn but has many online materials and resources. Flask is easy and straightforward and has a lot of online materials and resources, and Fast API, which is the easiest if you're starting with web development but has few online resources.
+
+#### Conclusion
+After looking at them, you might be able to choose which one works better for you. I would love to share my thoughts on each of them. First, Django is perfect if you want to build robust full-stack websites because it has several functionalities and works very well in production. Flask is ideal for Machine learning Engineers or developers who want to quickly prototype a web application and build APIs easily and quickly. At the same time, FastAPI is perfect if you're looking for speed or scalability. Based on what you want to use it for, you should choose, but all of them generally are good and have good market demand.
```

Comment:
"I would love to share my thoughts on each of them." we do not need this sentence - (because you should have been sharing your thoughts via the article). 

This conclusion sounds like a main paragraph. The article should state your thoughts then have supporting evidence to back that claim up 👍